ui-admin/src/study/kits/KitManager.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import { Tab, Tabs } from 'react-bootstrap'
+import { StudyEnvContextT } from 'study/StudyEnvironmentRouter'
+import KitEnrolleeSelection from './KitEnrolleeSelection'
+import KitList from './KitList'
+
+
+/** Top-level kit management screen. */
+export default function KitManager({ studyEnvContext }: { studyEnvContext: StudyEnvContextT }) {
+  const { study } = studyEnvContext
+  const [activeTab, setActiveTab] = useState<string | null>('enrollees')
+
+  return <div className="ParticipantList container pt-2">
+    <div className="row">
+      <div className="col-12">
+        <h2 className="h5">{study.name} Sample Kits</h2>
+        <Tabs
+          activeKey={activeTab ?? undefined}
+          mountOnEnter
+          unmountOnExit
+          onSelect={setActiveTab}
+        >
```

Comment:
These tabs need to be routable -- e.g `https://localhost:3000/ourhealth/studies/ourheart/env/sandbox/kits/byKit`
and 
`https://localhost:3000/ourhealth/studies/ourheart/env/sandbox/kits/byEnrollee`
or somesuch.

This can be done by making "activeTab" a parameter passed into KitManager from KitRouter

Reply:
Yes, this is probably the thing I was least comfortable about not addressing before opening the PR. Well, that and neglecting UI tests ☹️. In both cases, I decided to get the PR going instead of delaying it further. I should be able to fix this today.